### PR TITLE
Shivam/elastic

### PR DESCRIPTION
**Pull request by shivamag00**

### CODE REVIEW

File: src/components/lists/ArticleListElastic.vue [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        let bottomOfWindow = (document.documentElement.scrollTop + window.innerHeight) - document.documentElement.offsetHeight > -0.7;
+        console.log(document.documentElement.scrollTop)
+        console.log(document.documentElement.offsetHeight)
+        console.log(window.innerHeight)
+        if (bottomOfWindow) {
+          this.page+=1;
+          axios.get(this.apiUrl+'&per_page='+this.per_page+'&page='+this.page)
+              .then(response => {
+                for (let i = 0; i < response.data.result.length; i++)
+                  this.articles.push(response.data.result[i]);
```
Comment (nilmadhab) [inline src/components/lists/ArticleListElastic.vue]: use foreach loop

File: src/components/lists/ArticleListElastic.vue [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    ArticleCardElastic
+  },
+  props: [
+    'apiUrl'
+  ],
+  methods: {
+    fetchArticleList: function () {
+      this.per_page = 5;
+      this.page = 0;
+      axios.get(this.apiUrl+'&per_page='+this.per_page+'&page='+this.page)
```
Comment (nilmadhab) [inline src/components/lists/ArticleListElastic.vue]: declare a cons url variable

File: src/components/lists/ArticleListElastic.vue (hunk 1 above)
Comment (shivamag00) [reply src/components/lists/ArticleListElastic.vue]: done

File: src/components/lists/ArticleListElastic.vue [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    ArticleCardElastic
+  },
+  props: [
+    'apiUrl'
+  ],
+  methods: {
+    fetchArticleList: function () {
+      this.per_page = 5;
+      this.page = 0;
+      let url = this.apiUrl+'&per_page='+this.per_page+'&page='+this.page;
```
Comment (nilmadhab) [inline src/components/lists/ArticleListElastic.vue]: make it `${this.apiUrl}&per_page=${this.per_page}` format for better readability

File: src/components/lists/ArticleListElastic.vue (hunk 2 above)
Comment (shivamag00) [reply src/components/lists/ArticleListElastic.vue]: done

File: src/views/BackendElastic.vue [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+<script>
+import {ELASTIC_API_BASE_URL} from '/src/config.js';
+import ArticleListElastic from "@/components/lists/ArticleListElastic";
+
+export default {
+  name: "BackendHomePage",
+  data() {
+    return {
+      baseURL :  ELASTIC_API_BASE_URL+'?tag_names[]=backend&sort_by=hotness_score&sort_direction=desc&approved=&class_name=Article',
```
Comment (nilmadhab) [inline src/views/BackendElastic.vue:37]: make a function to get the url by passing the tag for reusability

**Pull request merged**